src/absil/il.fsi

```diff
@@ -349,7 +349,7 @@ and
     | TypeVar of uint16           
     /// Custom modifiers. 
     | Modified of            
-          /// True if modifier is "required" 
+          /// <c>true</c> if modifier is "required". 
```

Comment:
We don't tend to use ``<c>`` markup in the F# compiler codebase.  I prefer the plain text here. thanks